x-pack/plugins/endpoint/public/applications/endpoint/view/alerts/details/overview/index.tsx

```diff
@@ -4,91 +4,95 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import React, { memo, useMemo } from 'react';
+import styled from 'styled-components';
 import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n/react';
 import { EuiSpacer, EuiTitle, EuiText, EuiHealth, EuiTabbedContent } from '@elastic/eui';
 import { useAlertListSelector } from '../../hooks/use_alerts_selector';
 import * as selectors from '../../../../store/alerts/selectors';
+import { ResolverEvent } from '../../../../../../../common/types';
 import { MetadataPanel } from './metadata_panel';
 import { FormattedDate } from '../../formatted_date';
 import { AlertDetailResolver } from '../../resolver';
 
-export const AlertDetailsOverview = memo(() => {
-  const alertDetailsData = useAlertListSelector(selectors.selectedAlertDetailsData);
-  if (alertDetailsData === undefined) {
-    return null;
-  }
-  const selectedAlertIsLegacyEndpointEvent = useAlertListSelector(
-    selectors.selectedAlertIsLegacyEndpointEvent
-  );
+export const AlertDetailsOverview = styled(
+  memo(({ className }: { className?: string }) => {
+    const alertDetailsData = useAlertListSelector(selectors.selectedAlertDetailsData);
+    if (alertDetailsData === undefined) {
+      return null;
+    }
```

Comment:
Could you please remove the `selectedAlertIsLegacyEndpointEvent` selector if it's no longer used?

Reply:
done